src/states/node.rs

```diff
@@ -1651,18 +1575,6 @@ impl Node {
             }
         }
 
-        // Until we're established do not relay any messages.
-        let src = signed_msg.routing_message().src;
```

Comment:
Also not 100% clear how this now work.

Reply:
For non-established states, routing message sending goes via proxy and is handled by `NotEstablished::send_routing_message_via_proxy`. So the check that the sender is ourselves should have gone there. I'll add it.